R/fontconfig.R

```diff
@@ -64,3 +66,12 @@ fc_match_debug <- function(debug) {
   names(debug) <- "FC_DEBUG"
   debug
 }
+
+# Hack to force Fontconfig to reload debug flag
```

Comment:
That makes me a little nervous - I'm not sure reloading a DLL multiple times is going to be fragile or not.  Could you have a global debug flag and only reload the dll if it changes?  Maybe wrap up in a `set_debug()` helper?